### ISSUE TITLE
Feature Request: sync across devices

### ISSUE THREAD

**Issue by pgweiss** [kebernet/shortyz#83]
I use your app on my phone and two tablets. It would be nice to be able to work on a puzzle on the bus with my phone and continue later on my tablet.

I realize that you would need some storage to be able to implement this. You already allow us to present Google credentials for Google Games. What about also using those to save our state to Google Drive?

**Comment by kebernet** [comment]
Yeah, this has kind of been a todo for a while now. There are some complexities around it, though, that I haven't really worked through.
